src/google/protobuf/metadata.h

```diff
@@ -59,8 +59,7 @@ namespace internal {
 class LIBPROTOBUF_EXPORT InternalMetadataWithArena {
```

Comment:
Definition of this class is all in this header file. Would it work if we just remove LIBPROTOBUF_EXPORT?